AGENTS.md

```
@@ -0,0 +1,91 @@
+the notes below only cover non-obvious things needed to run it in a Cloud Agent.
+
+### Toolchain / Node
+- The repo requires **Node 24** (`.nvmrc`), but the VM's default `node`
+  (`/exec-daemon/node`) is Node 22 and sits early in `PATH`. Interactive/login
+  shells are already fixed to use Node 24 + pnpm 10.33.2 via `~/.bashrc`
+  (`nvm use 24` + a `PATH` prepend), so `node`/`pnpm` work normally in tmux and
+  new terminals. The update script's `pnpm install` works on either Node version
+  (there is no `engine-strict`), but **run the app/Convex with Node 24**.
+
+### Services (all must run for end-to-end use)
+| Service | How to start | Notes |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)